.ci/scripts/utils.sh

```diff
@@ -89,6 +90,17 @@ install_flatc_from_source() {
   popd || return
 }
 
+install_arm() {
```

Comment:
Nit: Since we are using relative paths, it might be a good ideas to make sure we are indeed in the right dir.